cmake/host-tools.cmake

```diff
@@ -7,6 +7,37 @@ find_program(
   )
 if(${WEST} STREQUAL WEST-NOTFOUND)
   unset(WEST)
+else()
+  # If west is found, make sure its version matches the minimum
+  # required one.
+  set(MIN_WEST_VERSION 0.5.4)
+  execute_process(
+    COMMAND
+    ${WEST} --version
+    OUTPUT_VARIABLE west_version_output
+    )
+
+  # The west --version output is currently multiline: bootstrapper
+  # followed by installation version. Test both. This will still work
+  # when .west/west is eliminated from the installation and obtained
+  # via pypi, which will result in a single line of output.
+  string(REGEX REPLACE "\n" ";" west_version_output "${west_version_output}")
+  set(west_version_status "")
```

Comment:
This doesn't seem to be used elsewhere?

Reply:
Sorry, that's cruft from an earlier version